_emails/us/michigan/royaloak.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Royal Oak, MI
+permalink: "/lasvegas"
+name: Letter to the City Commission and City Manager
+state: MI
+city: Royal Oak
+layout: email
+recipients:
+- ccfournier@romi.gov
+- ccdubuc@romi.gov
+- ccgibbs@romi.gov
+- cclevasseur@romi.gov
+- ccparuch@romi.gov
+- ccmacey@romi.gov
+- ccdouglas@romi.gov
+- citymanager@romi.gov
+subject: A budget for the community well-being of Royal Oak
+body: |-
+  To the City of Royal Oak's 50th Commission and City Manager,
+
+  My name is [YOUR NAME], and I am a resident of Royal Oak. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. As the Commission prepares the budget for the 2021 fiscal year beginning July 1, 2020, I demand that you reconsider the budget of the Royal Oak Police Department (ROPD).
+
+  The fourth goal of the ROPD listed on the 2020-2021 Recommended Budget is "to provide exceptional public safety for major events, such as Woodward Dream Cruise and Arts, Beats and Eats." Given that both of those events have been canceled it is reasonable to ask that the $61,000 that has been budgeted for these events be reallocated to sectors that could more effectively benefit Royal Oak, such as community development, health and welfare, affordable housing, the commissions' sustainability initiative, public works, and education.
+
+  The 2020-2021 Recommended Budget has proposed to increase to the ROPD budget by $391,090 with $15,000 specifically dedicated to weapons and ammunition. I emphatically condemn this proposal. ROPD is slated to receive $19,818,220, which is 31% of the proposed Personnel Services budget for the 2020-2021 fiscal year. The ROPD proposed budget is particularly startling compared to the $327,610 that community development is presently set to receive. As a resident of Royal Oak, I demand a budget that reflects a commitment to community well-being, rather than one that entitles the police forces that tear it apart.
```

Comment:
If the 19,818,220 amount represents total expenditures for police, then it might be better to use the total expenditure amount for community development (342,990) rather than just the personnel service expenditure (327,610) for a more equal comparison.

Reply:
Agreed!